keras_cv/__init__.py

```diff
@@ -18,6 +18,7 @@
 version_check.check_tf_version()
 # isort:on
 
+from keras_cv import callbacks
```

Comment:
From the example training script, it sounds like we're just re-import the callback, but from here it sounds like we're offering it as a public API?
@ianstenbit WDYT?

Reply:
any reason to not export it?  No matter what our users will want a way to evaluate COCO metrics using PyCOCOTools.  Some people care about exact numerics.

Reply:
Isn't the consensus to replace the callback with PyMetrics?
@ianstenbit will know more